mobile/lib/common/settings_screen.dart

```diff
@@ -202,7 +203,11 @@ class _SettingsScreenState extends State<SettingsScreen> {
         ),
         ElevatedButton(
             onPressed: () async {
-              var file = await FLog.exportLogs();
+              // TODO:
+              logger.toString();
+
+              // TODO: fix me
```

Comment:
Does exporting the logs still work with that framework?

Reply:
I tried to tackle it once upon a time, and couldn't get this to work easily, so instead went for a quickfix during Dart3 upgrade: #1092 .

if I remember correctly there was no API, but we should be able to read the stream into memory and export it

Reply:
Sorry, yes it works. The fix me is outdated.

Reply:
Can we remove it then? 🙂